src/main/java/me/jellysquid/mods/lithium/mixin/world/explosions/MixinExplosion.java

```diff
@@ -60,9 +60,7 @@
     public abstract DamageSource getDamageSource();
 
     @Shadow
-    public static float getExposure(Vec3d self, Entity entity) {
-        throw new UnsupportedOperationException();
-    }
+    public static native float getExposure(Vec3d self, Entity entity);
```

Comment:
yeah it is, I do that too sometimes because making an empty method stub every time gets a bit annoying, since mcdev doesn't autogenerate method stubs for static methods iirc